### PR TITLE
Fix merge conflicts

### PR DESCRIPTION
**Pull request by mubstimor**
### What this PR does?
Resolves conflicts that were introduced after merging ft-create-articles-162633502 to develop

### CODE REVIEW

**Comment by mycok** [comment]
@mubstimor  please make the following changes; 
* Remove the tag implementations (model, url, views, serialiser) from your merged branch since it's not part of your user story expected implementation.
* Remove the commented out tagList model field on the article model. 
* Remove the unique validator import from your serialiser module since you don't use it anywhere.

File: authors/apps/authentication/tests/base.py [hunk 1]
```diff
@@ -98,3 +111,29 @@ def verify_registered_user_account(self):
                 args=(token, user_email)
             ), format="json"
         )
+
+    def get_user2_acccount_verification_email(self):
+            userData = {
```
Comment (Qadriyah) [inline authors/apps/authentication/tests/base.py]: Use snake_case for the variable userData

File: authors/apps/articles/tests/test_articles.py [hunk 2]
```diff
@@ -135,5 +138,10 @@ def test_delete_article_by_invalid_author(self):
...
 def get_user_token(self):
-    user = User.objects.create_user(**self.user.get('user'))
-    return user.token
\ No newline at end of file
+    user = self.register_and_loginUser()
+    return user.data['token']
+
+def get_user2_token(self):
+    user = self.register_and_login_user2()
+    pprint(user.data)
```
Comment (Qadriyah) [inline authors/apps/articles/tests/test_articles.py]: Please remove this print statement, it's redundant!

File: authors/apps/articles/tests/test_articles.py [hunk 3]
```diff
@@ -88,9 +90,10 @@ def test_update_article_by_invalid_author(self):
         article_slug = request.data['slug']
         
         # create second user and update first user's article
-        user2 = User.objects.create_user(**self.user.get('user2'))
+        user2_token = get_user2_token(self)
+        pprint("WE HERE")
```
Comment (Qadriyah) [inline authors/apps/articles/tests/test_articles.py]: Remove this print statement, it's redundant! Remember to remove the import too

**Comment by mubstimor** [comment]
> @mubstimor please make the following changes;
> 
> * Remove the tag implementations (model, url, views, serialiser) from your merged branch since it's not part of your user story expected implementation.
> * Remove the commented out tagList model field on the article model.
> * Remove the unique validator import from your serialiser module since you don't use it anywhere.

resolved.

File: authors/apps/articles/tests/test_articles.py (hunk 2 above)
Comment (mubstimor) [reply authors/apps/articles/tests/test_articles.py]: resolved

**Comment by jeanjoe** [comment]
LGTM

**Pull request merged**